packages/core/botpress/src/cli/templates/init-default/botfile.js

```diff
@@ -80,8 +80,8 @@ module.exports = {
     Dialog Manager (DM)
   */
   dialogs: {
-    timeoutInterval: '2m',
-    janitorInterval: '10s'
+    timeoutInterval: '30m',
+    janitorInterval: '10m'
```

Comment:
Which item on the list of improvements is this one about?

Reply:
All improvements on the list have the same name as the commit message, so it is **Fix Dialog manager timeouts**

At the very least it fixes the bug of `janitorInterval` being ignored and bumping up the timeout interval just brings it in line with how sessions are managed in the de-facto analytics industry standards:
https://support.google.com/analytics/answer/2731565?hl=en

**ALSO**: I just noticed in the docs that timeout generates an event sent to `timeout` nodes

Have not tested this functionality and think it ought to be documented. Could timeout node "restart" the flow to prevent session expiration? Could it get access to the context, of where the flow was last? I don't believe actions receive context as a parameter at this time - this probably needs a fix...